Branch-SDK/src/io/branch/indexing/BranchUniversalObject.java

```diff
@@ -501,15 +501,56 @@ public void registerView(@Nullable RegisterViewStatusListener callback) {
 
     //--------------------- Create Link --------------------------//
 
-
+    /**
+     * Creates a short url for the BUO synchronously.
+     *
+     * @param context        {@link Context} instance
+     * @param linkProperties An object of {@link LinkProperties} specifying the properties of this link
+     * @return A {@link String} with value of the short url created for this BUO. A long url for the BUO is returned in case link creation fails
+     */
     public String getShortUrl(@NonNull Context context, @NonNull LinkProperties linkProperties) {
         return getLinkBuilder(context, linkProperties).getShortUrl();
     }
 
+    /**
+     * Creates a short url for the BUO synchronously.
+     * xa
+     *
+     * @param context          {@link Context} instance
+     * @param linkProperties   An object of {@link LinkProperties} specifying the properties of this link
+     * @param defaultToLongUrl A {@link Boolean} specifies if a long url should be returned in case of link creation error
```

Comment:
a bit nitpicky: the argument isn't a Boolean, but a primitive boolean (lowercase b).
Mentioned elsewhere as well